README.md

```diff
@@ -121,4 +121,6 @@ See  [CONTRIBUTING.md](CONTRIBUTING.md).
 
 # License
 
-Code and documentation Copyright 2023 Siemens AG
+Code and documentation under [MIT License](LICENSE.md)
```

Comment:
**LICENSE / LICENSE.md** . since we are having file type **LICENSE** in **main**.